pkg/cloud/gcp/actuators/machine/actuator.go

```diff
@@ -11,6 +11,10 @@ import (
 	controllerclient "sigs.k8s.io/controller-runtime/pkg/client"
 )
 
+const (
+	scopeFailMsg = "failed to create scope for machine %q: %v"
```

Comment:
I think this would be better as `scopeFailFmt` - then calls to `fmt.Errorf(scopeFailFmt, ...)` would read a lot better.